content/vendor/snapshots/snapshots.md

```diff
@@ -0,0 +1,164 @@
+---
+date: 2019-01-23
+linktitle: "Snapshots"
+title: Snapshots
+aliases:
+  - /vendor/snapshots/
+---
+
+## Requirements
+
+### Application YAML
+
+Enable snapshots in your app with the field `spec.allowSnapshots: true` in your kots.io/v1beta1 Application yaml.
+
+```
+apiVersion: kots.io/v1beta1
+kind: Application
+metadata:
+  name: app-slug
+spec:
+  title: App Name
+  allowSnapshots: true
+```
+
+### Customer License
+
+For each customer license there is an Allow Snapshots checkbox. Both the application yaml and the customer license must have snapshots enabled in order for the Admin Console to display snapshot UI to the user.
+
+### Velero
+
+The customer must have [velero](https://velero.io/) installed in the `velero` namespace along with [Restic](https://velero.io/docs/v1.2.0/restic/).
+
+#### Modify default install (this is going away)
+
+The following patches must be applied to the default velero install with the CLI.
+
+Load all provider plugins as initContainers. (Velero deployment only, not applicable to restic daemonset).
+
+[velero deployment](https://github.com/replicatedhq/kURL/blob/master/addons/velero/1.2.0/deployment.yaml#L20-L34)
+
+Mount secrets volumes in subdirectory
+
+[velero deployment](https://github.com/replicatedhq/kURL/blob/master/addons/velero/1.2.0/deployment.yaml#L48-L53)
+
+[restic daemonset](https://github.com/replicatedhq/kURL/blob/master/addons/velero/1.2.0/restic-daemonset.yaml#L20-L31)
+
+Use env vars to point to provider-specific credentials
+
+[velero deployment](https://github.com/replicatedhq/kURL/blob/master/addons/velero/1.2.0/deployment.yaml#L64-L69)
+
+[restic daemonset](https://github.com/replicatedhq/kURL/blob/master/addons/velero/1.2.0/restic-daemonset.yaml#L46-L51)
+
+Use provider-specific secret volumes
+
+[velero deployment](https://github.com/replicatedhq/kURL/blob/master/addons/velero/1.2.0/deployment.yaml#L71-L82)
+
+[restic daemonset](https://github.com/replicatedhq/kURL/blob/master/addons/velero/1.2.0/restic-daemonset.yaml#L68-L73)
+
+
+## Release YAML Configuration
+
+### Backup
+
+Kotsadm requires exactly one [velero.io/v1/Backup](https://velero.io/docs/v1.2.0/api-types/backup/) resource be included in the release yaml.
```

Comment:
Is this a strict requirement? Will new releases be broken without this? Or does the Admin Console require this for snapshot/restore capabilities?

Reply:
The Admin Console requires it.